src/helm/benchmark/presentation/schema.py

```diff
@@ -245,9 +247,9 @@ def __post_init__(self):
         self.name_to_run_group = {run_group.name: run_group for run_group in self.run_groups}
 
 
-def read_schema() -> Schema:
-    hlog(f"Reading schema from {SCHEMA_YAML_FILENAME}...")
-    schema_path = resources.files(SCHEMA_YAML_PACKAGE).joinpath(SCHEMA_YAML_FILENAME)
+def read_schema(filename: str) -> Schema:
+    schema_path = resources.files(SCHEMA_YAML_PACKAGE).joinpath(filename)
```

Comment:
Add a TODO to merge in model metadata from `model_metadata.yaml`.

Reply:
Done